spec/controllers/physical_storage_controller_spec.rb

```diff
@@ -14,6 +16,18 @@
     login_as FactoryBot.create(:user)
   end
 
+  it "compare end2end create physical storage hash" do
+    end2end_json_path = Rails.root.join("spec", "fixtures", "end2end", "physical_storage.json").to_s
```

Comment:
Where is this file?

Reply:
Oh I see it's in core....but what generates that file in core?

Reply:
We put it in core when creating the test that we want to pair together.
If the tests change significantly we'd need to change the file too.

Reply:
I removed the json in the core so it compared directly with the json in the provider. the core changes are no longer needed